Dockerfile_intel

```diff
@@ -215,16 +215,9 @@ COPY server server
 COPY server/Makefile server/Makefile
 ENV UV_SYSTEM_PYTHON=1
 RUN cd server && \
+    make gen-server && \
     pip install -U pip uv && \
-	uv sync --frozen --extra gen --extra accelerate --extra compressed-tensors --extra quantize --extra peft --extra outlines --no-install-project && \
-    . ./.venv/bin/activate && \
-    make gen-server-raw
-
-RUN cd server && \
-	uv sync --frozen --extra gen --extra accelerate --extra compressed-tensors --extra quantize --extra peft --extra outlines && \
-    . ./.venv/bin/activate && \
-    pwd && \
-    text-generation-server --help
+    uv pip install -e ".[accelerate, compressed-tensors, peft, outlines]" --no-cache-dir
```

Comment:
Not sure if intended, but missing `quantized` here

```suggestion
    uv pip install -e ".[accelerate, compressed-tensors, quantized, peft, outlines]" --no-cache-dir
```

Reply:
Should be `quantize` though :).

Reply:
Actually I added it in the previous PR inadvertently, this is what was on main before. I rolled back the locking for Intel pretty much.